overlord/servicestate/service_control_test.go

```diff
@@ -108,6 +109,224 @@ func (s *serviceControlSuite) mockTestSnap(c *C) {
 	c.Assert(err, IsNil)
 	err = ioutil.WriteFile(filepath.Join(dirs.GlobalRootDir, "etc/systemd/system/snap.test-snap.foo.service"), nil, 0644)
 	c.Assert(err, IsNil)
+
+	return inf
+}
+
+func verifyControlTasks(c *C, tasks []*state.Task, expectedAction, expectedFlag string, expectedServices ...string) {
+	// sanity, ensures test checks below are hit
+	c.Assert(len(tasks) > 0, Equals, true)
+	var i int
+	for i < len(tasks) {
+		var argv []string
+		if tasks[i].Kind() == "exec-command" {
+			switch expectedAction {
+			case "start":
+				if expectedFlag != "" {
+					c.Assert(tasks[i].Get("argv", &argv), IsNil)
+					c.Check(argv, DeepEquals, append([]string{"systemctl", "enable"}, expectedServices...))
```

Comment:
Perhaps use `expectedFlag` instead of the literal `"enable"` here. I would also rename the argument to `supportAction` or something similar.